packages/verticaltabs/src/react/item.tsx

```diff
@@ -9,91 +9,39 @@ import {
   omit,
   RefFor
 } from '@pluralsight/ps-design-system-util'
-import glamorDefault, * as glamorExports from 'glamor'
 import React, { useEffect } from 'react'
 
 import { useHideLabels } from './context'
-import stylesheet from '../css/index'
 import { List, CollapsibleList } from './list'
-
-type StyleFn = (
-  themeName?: ValueOf<keyof typeof Theme.names>,
-  props?: Record<string, unknown>
-) => glamorExports.StyleAttribute
-
-const glamor = glamorDefault || glamorExports
+import '../css/index.css'
+import classNames from 'classnames'
```

Comment:
Auto import bug? I think we want to use our own `classNames` fn from utils.

Reply:
looks like